app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,59 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit update)
+  before_action :valid_user, only: %i(edit update)
+  before_action :check_expiration, only: %i(edit update)
```

Comment:
```suggestion
  before_action :get_user, :valid_user, :check_expiration, only: %i(edit update)
```

ko nên bắt đầu tên method là set/get nhé thanh niên